src/sbl_filing_api/entities/repos/submission_repo.py

```diff
@@ -63,6 +63,14 @@ async def get_filing(session: AsyncSession, lei: str, filing_period: str) -> Fil
     return result[0] if result else None
 
 
+async def get_filings(session: AsyncSession, leis: list[str], filing_period: str) -> list[FilingDAO]:
+    stmt = select(FilingDAO).where(FilingDAO.lei.in_(leis)).filter_by(filing_period=filing_period)
```

Comment:
this works, but maybe we can modify the query_helper, or have a different query_helper that allows for chained `.filter()`, since in this context `.where()`, and `.filter()` does the same thing, so if were to use filter chain, this can become
```python
select(FilingDAO).filter(FilingDAO.lei.in_(leis)).filter(FilingDAO.filing_period == filing_period)
```
so if we were to do a diff version of query_helper, we'd be able to do
```python
query_helper(FilingDAO, FilingDAO.lei.in_(leis), FilingDAO.filing_period == filing_period)
```

Reply:
if I remember correctly, filter_by has a key = value limitation; in this scenario a where in clause is needed.